lib/jekyll/hooks.rb

```diff
@@ -88,19 +80,17 @@ def self.insert_hook(owner, event, priority, &block)
     end
 
     # interface for Jekyll core components to trigger hooks
-    def self.trigger(instance, event, *args)
-      owner_symbol = OWNER_MAP[instance.class]
-
+    def self.trigger(owner, event, *args)
```

Comment:
i think we can keep using instance, and just accept either a symbol or an instance of something.